core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -535,7 +538,7 @@ class SparkSubmitSuite
 
   test("resolves command line argument paths correctly") {
     val jars = "/jar1,/jar2"                 // --jars
-    val files = "hdfs:/file1,file2"          // --files
+    val files = "local:/file1,file2"          // --files
```

Comment:
It is kinda difficult to test download file from hdfs now, but we should cover this scene in the future.